app/models/driver.rb

```diff
@@ -3,7 +3,13 @@ class Driver < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
 
   devise :database_authenticatable, :rememberable, :trackable
-  validates :number, presence: true, uniqueness: true
-  validates_format_of :number, with: /\A(0)([0-9]{9})\z/
-
+  
+   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
@Kolisnyk please check indentation here.
@yelizaveta-dziuba please check if this regular expression is the same as you use on front end.